lib/resources/r4/financial_transaction_create.yaml

```diff
@@ -0,0 +1,325 @@
+---
+name: Financial Transaction
+field_name_base_url: http://hl7.org/fhir/r4/Basic
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  action: create
+  example: |
+    {
+      "resourceType":"Basic"
+    }
+  note: resourceType must be Basic
+  url: http://hl7.org/fhir/r4/json.html#resources
+
+- name: identifier
+  required: 'Yes'
+  cardinality: 0..*
+  type: Identifier
+  description: Identifier assigned to the resource for business purposes.
+  action: create
+  example: |
+    {
+      "identifier": [
+        {
+            "value": "33358000",
+            "use": "usual"
+        }
+      ]
+    }
+  note: At least one external transaction identifier must be provided.
+  url: http://hl7.org/fhir/basic-definitions.html#Basic.identifier
+
+- name: code
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  description: Identifies the 'type' of resource - equivalent to the resource name for other resources.
+  action: create
+  example: |
+    {
+      "code": "fintrans"
+    }
+  note: This will always be fintrans
+  url: http://hl7.org/fhir/r4/basic-definitions.html#Basic.code
+
+- name: financial-transaction-type
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: Category of transaction Payment Type Codes (Example), Whether this is a payment or adjustment.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-type",
+      "valueCode": "Adjustment"
```

Comment:
This needs to be updated to a CodeableConcept

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/0e8234a13abfe664bfb577f71a8bbc9a6bb9af88#diff-26870e1b94fc9955e130f3f940dbe2e7R58

Reply:
@ReaganYuan This still is not done

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/ad66ee297c2d981a2bce38832bfcbd6d32c5d086#diff-26870e1b94fc9955e130f3f940dbe2e7R61